release-team/role-handbooks/branch-manager/README.md

```diff
@@ -207,6 +251,17 @@ The cherry pick script is also fairly self documenting in terms of how to invoke
 
 There has been quite a bit of recent discussion (see: [1](https://github.com/kubernetes/community/pull/2408), [2](https://github.com/kubernetes/community/pull/1980)) around improving both the cherry pick process process and its documentation.  Watch for and contribute to improvements.
 
+
+In the time between code thaw and the official release (when the patch release team takes over) consider the following when assessing the cherry-picks:
+- Check regularily if there are new cherry picks with [`is:open is:pr base:release-1.14 label:do-not-merge/cherry-pick-not-approved`][cherry-pick-query]
+- Consider that each cherry-pick diverges `rc.1` (or the latest release candidate) from the bits to be released as the official release
```

Comment:
minor nit: remove _or_ from _(or the latest release candidate)_

Reply:
We had releases where we cut more then on `rc.1`. In those cases the last release candidate could be `rc.3`. I will rephrase to make that clearer.